project/Dependencies.scala

```diff
@@ -26,7 +26,7 @@ object Dependencies {
   val mavenApiVersion = "3.6.1"
   val mavenAnnotationsVersion = "3.5"
   val mavenScalaPluginVersion = "3.2.2"
-  val gradleVersion = "3.0"
+  val gradleVersion = "3.5"
```

Comment:
Any reason why we're doing this? We want to be compatible with previous Gradle versions and test as old versions as possible, so that's why we use 3.0 here.

Reply:
It's because of the fork options, apparently they were introduced in 3.5. I think it's reasonable to upgrade API to 3.5, we still support 3 this way and I doubt that anybody is using 3.0.0 anyway.